components/Navigation/Navigation.test.js

```diff
@@ -0,0 +1,12 @@
+import "@testing-library/jest-dom";
+import { getByRole, render } from "@testing-library/react";
+import Navbar from ".";
+
+const children = <li>Foo</li>;
+
+test("Navbar", async () => {
+  // wait for component to render
+  render(<Navbar>{children}</Navbar>);
```

Comment:
Since you are only using it once, I would pass the `children` prop directly:
```suggestion
  render(<Navbar><li>Foo</li></Navbar>);
```

Reply:
You're absolutely right - thank you very much! :)